content/faq/how-to-clean-install-preserve

```diff
@@ -0,0 +1,57 @@
+---
+title: How do I do a clean install of LBRY while preserving important data?
+category: setup
+---
+
+Under certain circumstances, particularly if you've tested earlier beta versions, it may be recommended that you perform a clean installation of LBRY.  This generally involves uninstalling the current LBRY application, removing various directories/files and then reinstalling the latest version of LBRY. Different operating systems use different folder locations and [this post on directories](https://lbry.io/faq/lbry-directories) will help if you cannot locate the right files or folders.  
```

Comment:
Added some light clean up to this file. Most importantly, please always link email addresses and URLs, and typically URLs should be entered with a label rather than entered directly.